media/css/base/footer-refresh.scss

```diff
@@ -0,0 +1,614 @@
+                    min-width: 100%;
+                }
+            }
+        }
+
+        .mzp-c-newsletter-details {
+            max-width: 100%;
+
+            @media(min-width: $screen-lg) {
+                display: grid;
+                grid-template-columns: 1fr 1fr;
+                grid-template-rows: 1fr 1fr;
+                column-gap: $moz24-space-lg;
+            }
+
+            p {
+                margin-bottom: 0;
+                line-height: 1.3;
+                font-weight: 600;
+                font-family: $primary-font;
+                text-align: start;
+
+                @media(min-width: $screen-lg) {
+                    display: inline-grid;
+
+                    &:first-of-type {
+                        grid-column-start: 1;
+                        grid-column-end: 2;
+                    }
+
+                    &:nth-of-type(2) {
+                        grid-column-start: 2;
+                        grid-column-end: 3;
+                    }
+                }
+
+                &:has(> label.mzp-u-inline) {
+                    margin-bottom: $moz24-space-lg;
+
+                    @media(min-width: $screen-lg) {
+                        grid-column-start: 1;
+                        grid-column-end: 3;
+                    }
+
+                    input {
+                        margin: 0;
+                    }
+                }
+            }
+
+            select {
+                font-family: $primary-font;
+                background-color: $color-primary-gray;
+                border: 2px solid $color-white;
+                color: $color-tertiary-gray;
+                width: 100%;
+                background-position-x: 100%;
+                background-position-y: 9px;
+                background-image: url('/media/img/logos/mozilla/footer-refresh/icon-arrow-down.svg');
+                background-repeat: no-repeat;
+
+                @media(min-width: $screen-lg) {
+                    min-width: 100%;
+                }
+            }
+
+            label:not(.mzp-u-inline) {
+                display: none;
+            }
+
+            .mzp-u-inline {
+                display: inline;
+
+                input {
+                    width: 13px;
+                    height: 13px;
+                    min-width: unset;
+                    line-height: 1;
+                }
+            }
+
+            a:visited {
+                color: $color-white;
+            }
+        }
+
+        .mzp-c-form-submit button {
+            border: 2px solid $color-zilla-green;
+            border-radius: 0;
+            font-family: $secondary-font;
+            position: relative;
+
+            &::after {
+                display: block;
+                content: "";
+                position: absolute;
+                width: calc(100% - 48px);
+                height: 1px;
+                left: 24px;
+                bottom: 8px;
+                background: $color-white;
+            }
```

Comment:
I agree. The current primary button designs look like typical links and the secondary button designs look like a combination of button and link. I can remove the underlines from the secondary buttons to make them look more like button for now. Hopefully this will be addressed from the design level soon